### PR TITLE
Add VTK - becomes dmriqc compatible 

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,96 +1,191 @@
...
+          -DOSMESA_LIBRARY=/usr/lib/libOSMesa.so \
+          -DCMAKE_INSTALL_PREFIX=/usr/ ../VTK-src/VTK*
+
+RUN  make -j 8 &&\
+     make install
+
+RUN mv /usr/bin/bk_python2 /usr/bin/python2
+RUN mv /usr/bin/bk_python /usr/bin/python
+
+RUN vtkpython --version
```
Comment (GuillaumeTh) [inline Dockerfile]: You can remove this here.

File: Dockerfile [hunk 2]
```diff
@@ -1,96 +1,191 @@
...
+RUN update-alternatives --config python3 &&\
+    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 1 &&\
+    update-alternatives  --set python3 /usr/bin/python3.7
+
+ENV PYTHON_INCLUDE_DIR=/usr/include/python3.7
+ENV PYTHON_LIBS=/usr/lib/python3.7/config-3.7m-x86_64-linux-gnu/libpython3.7.so
+ENV PYTHON_LIBRARY=/usr/lib/python3.7/config-3.7m-x86_64-linux-gnu/libpython3.7.so
+RUN pip3 install -U setuptools
+
+RUN mv /usr/bin/python2 /usr/bin/bk_python2
```
Comment (GuillaumeTh) [inline Dockerfile]: Is it not easiest to directly use python3.7 as python and just ignore python2 ?

Comment (arnaudbore) [reply Dockerfile]: Tried but didnt want to break anything I think this is a very small detail and won't change anything.

**Comment by CHrlS98** [comment]
Tested through https://github.com/scilus/containers-scilus/pull/10 .

Both `tractoflow` and `dmriqc_flow` work properly on my end locally and on a cluster!

**Pull request merged**